lib/improver/tests/test_set_up_test_cubes.py

```diff
@@ -361,6 +361,15 @@ def test_relative_to_threshold(self):
         self.assertEqual(len(result.attributes), 1)
         self.assertEqual(result.attributes['relative_to_threshold'], 'below')
 
+    def test_relative_to_threshold_set(self):
+        """Test that an error is raised if the "relative_to_threshold"
+        attribute has not been set when setting up a probability cube"""
+        data = np.flipud(self.data)
```

Comment:
We don't need this np.flipud as we are not trying to invert the relative_to_threshold attribute, just set it to none.

Reply:
oops - removed